src/urllib3/contrib/socks.py

```diff
@@ -66,23 +67,23 @@
 try:
     import ssl
 except ImportError:
-    ssl = None
+    ssl = None  # type: ignore
 
 
 class SOCKSConnection(HTTPConnection):
     """
     A plain-text HTTP connection that connects via a SOCKS proxy.
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Let's add an explicit `_sock_options` parameter to the signature. We can also use `typing.TypedDict` to make `_sock_options` really explicit.